orchagent/pfcwdorch.cpp

```diff
@@ -108,6 +111,27 @@ PfcWdAction PfcWdOrch<DropHandler, ForwardHandler>::deserializeAction(const stri
 }
 
 template <typename DropHandler, typename ForwardHandler>
+string PfcWdOrch<DropHandler, ForwardHandler>::serializeAction(const PfcWdAction& action)
```

Comment:
Plugins don't have to know which action type it is. It's orchagent's responsibility to make such decision.

Reply:
As above comments, detect plugin need to detect stormed queues in alert mode instead of restore plugin. This logic need to be added into lua script based on current implementation.